recipes/recipes_emscripten/sparrow-ipc/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  version: 0.2.0
+
+package:
+  name: sparrow-ipc
+  version: ${{ version }}
+
+source:
+  url: https://github.com/QuantStack/sparrow-ipc/archive/${{ version }}.tar.gz
+  sha256: 95658f58a99f05d311dc4382a7d9f3d5a6dc0f2c15148d93086589dec7057333
+  patches:
+  - patches/0001-use-host-flatc.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+  - ${{ compiler("cxx") }}
+  - cmake
+  - flatbuffers >=25.9.0,<25.10.0
+  host:
+  - flatbuffers >=25.9.0,<25.10.0
+  - sparrow
+  run:
+  - flatbuffers >=25.9.0,<25.10.0
+  - sparrow
```

Comment:
is this needed to add sparrow? If so, that indicates a missing run export of sparrow

Reply:
same for flatbuffers